helpers/teams/utils.js

```diff
@@ -165,32 +251,18 @@ async function createNewRootCollection(apollo, title, id){
 
 }
 
-async function exitFromTeam(apollo, id){
-    let response = undefined
-    while (true){
-        response = await apollo.mutate({
-            mutation: gql`
-              mutation($teamID: String!) {
-                deleteTeam(teamID: $teamID)
-              }
-            `,
-            variables: {
-              teamID: id,
-            },
-        })
-        if (response != undefined) break;
-    }
-    return response
-
-}
 
 export default {
     rootCollectionsOfTeam: rootCollectionsOfTeam,
     getCollectionChildren: getCollectionChildren,
     getCollectionRequests: getCollectionRequests,
-    editFolderForChildCollections: editFolderForChildCollections,
+    renameCollection: renameCollection,
     addChildCollection: addChildCollection,
-    deleteChildCollection: deleteChildCollection,
+    deleteCollection: deleteCollection,
     createNewRootCollection: createNewRootCollection,
-    exitFromTeam: exitFromTeam
+    createTeam: createTeam,
+    addTeamMemberByEmail: addTeamMemberByEmail,
+    renameTeam: renameTeam,
+    deleteTeam: deleteTeam,
+    exitTeam: exitTeam
```

Comment:
Since named exports have same name that of the function, I see no point in using named exports. Anyways, this is negligible.

This too should work:

```
export default {
    rootCollectionsOfTeam,
    getCollectionChildren,
    ...
    ...
    createTeam,
    addTeamMemberByEmail,
    renameTeam,
    deleteTeam,
    exitTeam
}
```

Reply:
I see, okay. Will update in the next PR.